packages/in_app_purchase/in_app_purchase_android/lib/src/in_app_purchase_android_platform_addition.dart

```diff
@@ -146,4 +146,15 @@ class InAppPurchaseAndroidPlatformAddition
       (BillingClient client) => client.isFeatureSupported(feature),
     );
   }
+
+  /// Returns Play billing country code based on ISO-3166-1 alpha2 format.
+  ///
+  /// See: https://developer.android.com/reference/com/android/billingclient/api/BillingConfig
+  /// See: https://unicode.org/cldr/charts/latest/supplemental/territory_containment_un_m_49.html
+  Future<String> getBillingConfig() async {
+    return _billingClientManager.runWithClientNonRetryable(
```

Comment:
Should this return a `BillingConfigWrapper` and then be also be retryable, as it returns a subclass of `HasBillingResponse`? 
https://github.com/flutter/packages/blob/cbe810031b883fcb7f602c8ecf8af05b7cf75905/packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_manager.dart#L92

Reply:
I wanted to keep the exposed api as small as possible so I thought the country code was the right level to expose. I didnt even think about if it should be retryable I just mirrored the other commands. Let me look at what that does and make a call.

Reply:
I think this comment still applies